backend/tests/core/validators_test.py

```
@@ -0,0 +1,103 @@
+
+    @pytest.mark.parametrize(
+        ("query"),
+        [
+            ("query_name"),
+            ("query-name"),
+            ("query name")
+        ]
+    )
+    def test_valid_query(self, query):
+        validate_query(query)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)